From 4238aaabf5b29c0ed27cc4e0cb00a0b5b1c9c97d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Victor=20Matar=C3=A9?= <matare@fh-aachen.de>
Date: Fri, 20 Mar 2020 18:00:47 +0100
Subject: [PATCH 1/2] tests: conditionalize SQL tests on embedded MYSQL

They all explicitly use the embedded mysql implementation, so they need
to be disabled if WITH_MYSQL_EMBEDDED is false.
---
 tests/core-impl/collections/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/core-impl/collections/CMakeLists.txt b/tests/core-impl/collections/CMakeLists.txt
index 42ffb0c3a2..b7e707dccb 100644
--- a/tests/core-impl/collections/CMakeLists.txt
+++ b/tests/core-impl/collections/CMakeLists.txt
@@ -1,3 +1,5 @@
 add_subdirectory( aggregate )
-add_subdirectory( db/sql )
+if( BUILD_MYSQLE_COLLECTION )
+	add_subdirectory( db/sql )
+endif()
 add_subdirectory( support )
-- 
2.25.2

